python/lsst/daf/butler/configs/storageClasses.yaml

```diff
@@ -184,8 +184,14 @@ storageClasses:
     pytype: lsst.verify.Measurement
   Stamps:
     pytype: lsst.meas.algorithms.Stamps
+    parameters:
+      - bbox
+      - origin
```

Comment:
The reader seems to only do something with the `bbox` parameter.  What am I missing?

Reply:
Hah! Thanks for asking. I'd just stupidly copied from Image. Removed and tested!